src/main/java/com/amazon/opendistroforelasticsearch/sql/parser/ElasticSqlExprParser.java

```diff
@@ -120,7 +120,7 @@ protected SQLExpr methodRest(SQLExpr expr, boolean acceptLPAREN) {
             return primaryRest(methodInvokeExpr);
         }
 
-        throw new ParserException("not support token:" + lexer.token());
+        throw new ParserException("Syntax error:" + lexer.token());
```

Comment:
Same here to throw `SyntaxAnalysisException` for consistency.

Reply:
Done, thanks!